iceoryx_examples/README.md

```diff
@@ -15,6 +15,7 @@
 |[iceensemble](./iceensemble/)               | Using multiple publishers for one topic                                   | :star::star:       |
 |[singleprocess](./singleprocess/)           | Communicating in a single process between threads                         | :star::star:       |
 |[user_header](./user_header/)               | Using a user-header for additional meta-information like timestamps       | :star::star:       |
+|[icediscovery](./icediscovery)              | Searching for specific services                                           | :star::star:       |
```

Comment:
I tend to keep the old title. Yes, the term `service` can be ambiguous but since we use it in many places (`ServiceDescription`, `ServiceDiscovery`, `findService`, `ServiceContainer`, ...) we should later decide if we want to rename all occurrences. I would find it confusing when the title states that we are searching publishers, but I have to create a `ServiceDiscovery` object and use the `findService` method.

Reply:
@budrus What is your opinion? Also considering #1136

Reply:
It's ok for me to stick with the title that talks about 'services'. Please change it to the same title used in `doc/website/examples/icediscovery.md` which is `Searching for currently available services`.